x-pack/platform/packages/shared/index-lifecycle-management/index_lifecycle_management_common_shared/src/locator.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { SerializableRecord } from '@kbn/utility-types';
+
+export interface IlmLocatorParams extends SerializableRecord {
+  page: 'policies_list' | 'policy_edit' | 'policy_create';
+  policyName?: string;
+}
```

Comment:
I'm not sure that's the right place - it looks like this is just for the top level app routing, not for specific locators within these apps.

As not all management apps are owned by the same team, I would like to keep this in a place owned by stack management as they will probably change the locator and the app itself at the same time.

But happy to go with what @elastic/kibana-management prefers here. If you want to go with a more elaborate solution I can also duplicate and inline the type for now and leave a todo, as it's more of a side quest of this PR

Reply:
Agree, the comment was not meant to be a blocker but to avoid switching visibility on a plugin that would then be mistakenly located in the new dir structure. I'll defer to what the Kibana team feels is best for this case, not a blocker on my end.

Reply:
I'd like to avoid changing the visibility of this plugin although at the moment I'm not sure of the best way to do that.

Reply:
Actually, I think there is an easier way - check the latest version of this. There is a shared package under `x-pack/platform/packages/shared/index-lifecycle-management` already, I moved the type over there.